### PR TITLE
Increase timeout for all amp-facebook tests for flakieness

### PR DESCRIPTION
**Pull request by mkhatib**
Just taking this up a notch for all FB elements. Previously increased for a single test case #2904

### CODE REVIEW

**Comment by jridgewell** [comment]
Oh, the irony of it flaking again. 😅

**Comment by mkhatib** [comment]
argh. You think FB is just blocking our travis IPs or something?

File: extensions/amp-facebook/0.1/test/test-amp-facebook.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ import {adopt} from '../../../../src/runtime';
 
 adopt(window);
 
-describe('amp-facebook', () => {
+describe('amp-facebook', function() {
```
Comment (cramforce) [inline extensions/amp-facebook/0.1/test/test-amp-facebook.js]: This is fine, but I think we should consider, uhm, reducing the test coverage here :)

Comment (mkhatib) [inline extensions/amp-facebook/0.1/test/test-amp-facebook.js]: yeah. agreed. I think overall third party rendering seems flakey so maybe worth visiting to mock out of the tests.

**Comment by sriramkrish85** [comment]
File a bug against yourself to do a permanent fix with proper stubs

**Comment by mkhatib** [comment]
Done #3037

**Pull request merged**